### DISCUSSION TITLE
Installing openCL

### DISCUSSION THREAD

**Discussion by vulkomilev** [artyom-beilis/pytorch_dlprim#89] [Q&A]
How to install correctly openCL on Ubuntu 24.04 with amd 5950 for this project ? 
I have tried several approaches but nothing worked.

**Comment by artyom-beilis** [comment]
I don't really understand what do you mean "install correctly OpenCL"

AMD 5950 is CPU. OpenCL/pytorch opencl backend is for GPU. Which GPU do you have?
